hangman/game.py

```diff
@@ -0,0 +1,82 @@
+from word_generator import WordGenerator
+from constants import *
+from hangman import Hangman
+
+
+class Game:
+    def __init__(self):
+        self.word_generator = WordGenerator(SOWPODS)
+        self.word_generator.fetch_data()
+        self.hangman = None
```

Comment:
hangman object can be injected via constructor. This makes unit testing easier when there are lots of dependencies for a class. 
then we don't need to initialise each of the dependency. we can just mock

also if there's a variant of hangman in future, lets say Hangman2, you can inject its object as well without changing the Game code
open close principle